src/commands/backend/logs.ts

```diff
@@ -1,9 +1,9 @@
 import { Command } from '@oclif/core'
-import { Jamsocket } from '../jamsocket'
+import { Jamsocket } from '../../jamsocket'
 
 export default class Logs extends Command {
     static description = 'Stream logs from a running backend.'
-
+    static aliases = ['logs']
```

Comment:
so this alias applies to the top-level command `logs`? as in `jamsocket logs`?

LGTM if so!

Reply:
Yup!